src/styles/blocks/header.scss

```diff
@@ -0,0 +1,104 @@
+body,
+h1,
+ul,
+li,
+p,
+a {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: $font-family;
+  background-color: #f9f9f9;
+  width: 100%;
+}
+
+.header {
+  width: 100%;
+  max-width: $max-width-container;
+  height: $header-height;
+  display: flex;
+  justify-content: space-between;
+  padding-left: $padding-container;
+  padding-right: $padding-container;
+  align-items: center;
+  background-color: #fff;
+  position: fixed;
+  font-size: $font-size-base;
+  margin: 0 auto;
+  box-sizing: border-box;
+  font-weight: $font-weight-base;
+
+  &__logo {
+    img {
+      height: 40px;
+      width: auto;
+      display: block;
```

Comment:
It's better to avoid using tag selectors directly in BEM methodology. Use classes instead.